API/NoDocs/Microsoft.Extensions.Telemetry.Tracing/SamplingOptions.cs

```diff
@@ -0,0 +1,16 @@
+// Assembly 'Microsoft.Extensions.Telemetry'
+
+using System.ComponentModel.DataAnnotations;
+using System.Runtime.CompilerServices;
+
+namespace Microsoft.Extensions.Telemetry.Tracing;
+
+public class SamplingOptions
```

Comment:
Can we remove this? I think it is dead code without an OTel dependency for distributed trace configuration.

Reply:
Done.